tdrs-backend/tdpservice/users/api/authorization_check.py

```diff
@@ -64,7 +64,7 @@ def get(self, request, *args, **kwargs):
         user = request.user
 
         user_in_valid_group = user.is_ofa_sys_admin
-
+        return HttpResponse(status=200)
```

Comment:
TODO: remove this after reviews are done, I added this to make the test easier, otherwise we had to create user first